### PR TITLE
Adding playlist management. Delete playlist not responding.

### PR DESCRIPTION
**Pull request by yurigava**

### CODE REVIEW

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js [hunk 1]
```diff
@@ -32,23 +33,45 @@ const PlaylistManagementPage = (props) => {
...
+        )
+        .then((response) => {
+          playlist.selected = !playlist.selected
+          setCollaborativePlaylists(() => playlists)
+        })
+        .catch((err) => console.error('Something went wrong'))
+    }
+  }
+
+  const forceReorder = () => {
```
Comment (TheMulittle) [inline src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: Can we rename this method to forceReorderClickedHandler? Just to follow the convention to specify handler methods

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js [hunk 2]
```diff
@@ -32,23 +33,45 @@ const PlaylistManagementPage = (props) => {
...
+        })
+        .catch((err) => console.error('Something went wrong'))
+    } else {
+      axios
+        .post(
+          `${process.env.REACT_APP_API_BASE_URL}/playlist/${playlistId}`,
+          { withCredentials: true },
+        )
+        .then((response) => {
+          playlist.selected = !playlist.selected
```
Comment (TheMulittle) [inline src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: isn't it better to use 'true' as it was before?

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js [hunk 3]
```diff
@@ -32,23 +33,45 @@ const PlaylistManagementPage = (props) => {
...
+    const playlist = playlists.find((playlist) => playlist.id === playlistId)
+
+    if(playlist.selected) {
+      axios
+        .delete(
+          `${process.env.REACT_APP_API_BASE_URL}/playlist/${playlistId}`,
+          { withCredentials: true },
+        )
+        .then((response) => {
+          playlist.selected = !playlist.selected
```
Comment (TheMulittle) [inline src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: isn't it better to use 'false' to clearly define that we want the button not to be selected (increases readability, I think)?

**Review by TheMulittle — CHANGES_REQUESTED** [review]

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js (hunk 1 above)
Comment (yurigava) [reply src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: Sure. I did not notice the convention. That makes sense.

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js (hunk 2 above)
Comment (yurigava) [reply src/pages/PlaylistManagementPage/PlaylistManagementPage.js:57]: I implemented the register/unregister logic. So clicking a playlist that is already registered will send the request to unregister the playlist and will remove the green border to tell the user that the playlist is no longer registered.

Comment (yurigava) [reply src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: Sure. Do you think there is a way to implement this without so much code duplication? The only thing that changes is the `post` and `delete` part. Is there a way to change the method with an if? Maybe some way to set the same `.then().catch()` on a promise created inside the if?

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js (hunk 1 above)
Comment (yurigava) [reply src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: Done.

File: src/pages/PlaylistManagementPage/PlaylistManagementPage.js (hunk 3 above)
Comment (yurigava) [reply src/pages/PlaylistManagementPage/PlaylistManagementPage.js]: This will just flip the selected state of the current playlist.

**Review by TheMulittle — APPROVED** [review]

**Pull request merged**